### ISSUE TITLE
Windows build fails because of Omnirig

### ISSUE THREAD

**Issue by radio-miskovice** [bazineta/js8call#2]
Make on Windows fails "You need to install Omnirig on this computer".

Environment: Win 10 Pro, MSYS2 installed, selected UCRT2 environment. Hamlib built successfully, however JS8Call build complains about Omnirig not installed.

I would prefer to have an option to omit Omnirig from build in order to unblock Windows build. 
From my own point of view I would like to exclude Omnirig at all, never heard of it before. Supposedly the latest version is from 2019, so it is not maintained anymore. If anything, I prefer FLrig which has the newest updates (e.g. QRP Labs' QMX latest fw version).

**Comment by Joe-K0OG** [comment]
I agree that for me Omnirig is of no value, but as I understand it can not be removed because some operators use other software that requires Omnirig to be present in WSJTX and JS8Call.

73,
-Joe-
K0OG

**Comment by radio-miskovice** [comment]
Joe, I agree with your view. I added a modest mod in the Cmake file so that if you run CMake with -D SKIP_OMNIRIG=1 (or any other value that evakuates to TRUE in if condition), you don't have to have Omnirig installed on your host Windows system. Basically, the build is then the same as on non-Windows system. This is what I need to build on my system if I don't want to spend time looking for what Omnirig is, how to install it, etc. It anyways seems to divert from the usual logic of GNU packages, where you install libXXXX to use it and libXXXX-dev if you want to build with it. So avoiding Omnirig saves me a pile of grey hair :)

Unfortunately, there are other obstacles that make Windows build difficult. I had resolved a number of them but still don't have a successful build - the last issue so far was that linker could not bind some network function (GetAdaptersAddresses). I am compiling under MSYS2, UCRT64 environment. Hopefully this is the last one and hopefully I will be able to resolve it soon.
